src/request.rs

```diff
@@ -55,8 +57,9 @@ impl fmt::Display for HttpDispatchError {
     }
 }
 
-impl From<HyperError> for HttpDispatchError {
-    fn from(err: HyperError) -> HttpDispatchError {
+// I'm not sure what this conversion is for.
```

Comment:
The `HttpDispatchError` trait allows error composition in `DispatchSignedRequest::dispatch()`